presto-native-execution/presto_cpp/main/QueryContextManager.cpp

```diff
@@ -93,8 +64,9 @@ std::unordered_map<std::string, std::string> toVeloxConfigs(
   // Use base velox query config as the starting point and add Presto session
   // properties on top of it.
   auto configs = BaseVeloxQueryConfig::instance()->values();
+  SessionProperties sessionProperties;
   for (const auto& it : session.systemProperties) {
-    configs[toVeloxConfig(it.first)] = it.second;
+    configs[sessionProperties.toVeloxConfig(it.first)] = it.second;
```

Comment:
Only configs defined by `BaseVeloxQueryConfig` are used.  Please keep `SessionProperties` as the source of truth, to ensure unified default values in the Prestissimo are used by Velox.